sync.py

```diff
@@ -0,0 +1,95 @@
+import json
+import requests
+import yaml
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+
+LOKI_GATEWAY_URL = 'http://loki-gateway.glueops-core-loki.svc.cluster.local'
+LOKI_GATEWAY_PORT = '80'
+LOKI_ENDPOINT = f'{LOKI_GATEWAY_URL}:{LOKI_GATEWAY_PORT}'
+LOKI_POST_HEADERS = {"Content-Type": "application/yaml"}
```

Comment:
if you drop the port then you can probably drop `:{LOKI_GATEWAY_PORT}`

Reply:
let me see if it works without the port

Reply:
you're totally right - I don't need to declare the port

nice!

updating